docs/source/markdown/podman-update.1.md.in

```diff
@@ -0,0 +1,59 @@
+% podman-update(1)
+
+## NAME
+podman\-update - Updates the cgroup configuration of a given container
+
+## SYNOPSIS
+**podman update** [*options*] *container*
+
+**podman container update** [*options*] *container*
+
+## DESCRIPTION
+
+Updates the cgroup configuration of an already existing container. The currently supported options are a susbet of the
+podman create/run resource limits options. These new options are non-persistent and only last for the current execution of the container; the configuration will be honored on its next run. This command takes one argument, a container name or ID, alongside the resource flags to modify the cgroup.
```

Comment:
I have troubles understanding the text above.  Can I apply it to a running container or will the changes only "be honored on its next run"?  Can you clarify that a bit more?